cloud/azure/storage-account-capacity/variables.tf

```diff
@@ -0,0 +1,50 @@
+# Module specific
+
+# storage_capacity detectors
+
+variable "storage_capacity_disabled" {
+  description = "Disable all alerting rules for storage_capacity detector"
+  type        = bool
+  default     = null
+}
+
+variable "storage_capacity_disabled_critical" {
+  description = "Disable critical alerting rule for storage_capacity detector"
+  type        = bool
+  default     = null
+}
+
+variable "storage_capacity_disabled_major" {
+  description = "Disable major alerting rule for storage_capacity detector"
+  type        = bool
+  default     = null
+}
+
+variable "storage_capacity_notifications" {
+  description = "Notification recipients list per severity overridden for storage_capacity detector"
+  type        = map(list(string))
+  default     = {}
+}
+
+variable "storage_capacity_aggregation_function" {
+  description = "Aggregation function and group by for storage_capacity detector (i.e. \".mean(by=['host'])\")"
+  type        = string
+  default     = ".mean(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
+}
+
+variable "storage_capacity_timer" {
+  description = "Evaluation window for storage_capacity detector (i.e. 5m, 20m, 1h, 1d)"
+  type        = string
+  default     = "12h"
+}
+
+variable "storage_capacity_threshold_critical" {
+  description = "Critical threshold for storage_capacity detector (in octets)"
+  type        = number
```

Comment:
it is the good choice if you want force user to set appropriate max for NBH alerting, is that what you want?

it is abviously possible to lower severity or disable this rule by default.